pkgs/development/python-modules/rouge-score/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, fetchPypi
+, fetchFromGitHub
+, python
+, buildPythonPackage
+, absl-py
+, nltk
+, numpy
+, six
+, pytest
+, pythonOlder
+}:
+let
+  testdata = fetchFromGitHub {
+    owner = "google-research";
+    repo = "google-research";
+    sparseCheckout = [ "rouge/testdata" ];
+    rev = "1d4d2f1aa6f2883a790d2ae46a6ee8ab150d8f31";
+    hash = "sha256-ojqk6U2caS7Xz4iGUC9aQVHrKb2QNvMlPuQAL/jJat0=";
+  };
+in buildPythonPackage rec {
+  pname = "rouge_score";
+  version = "0.1.2";
+  format = "setuptools";
+  disabled = pythonOlder "3.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    extension = "tar.gz";
+    hash = "sha256-x9TaJoPmjJq/ATXvkV1jpGZDZm+EjlWKG59+rRf/DwQ=";
+  };
+
+  patchPhase = ''
+    cp ${testdata}/rouge/test_util.py .
+    substituteInPlace test_util.py \
+      --replace 'os.path.join(os.path.dirname(__file__), "testdata")' '${testdata}/rouge/testdata'
+  '';
+
+  nativeBuildInputs = [ absl-py nltk numpy six ];
+
+  enableParallelBuilding = true;
+
+  doCheck = true;
+
+  checkPhase = ''
+    runHook preCheck
+    pushd "$out"
+    ${python.interpreter} lib/python3.10/site-packages/rouge_score/test_util.py
```

Comment:
This `test_util.py` and the one in lines 34-35 are different.
Therefore, the test does not work correctly.